plugins/awards/src/components/AwardEditComponent/AwardEditComponent.tsx

```diff
@@ -103,24 +101,8 @@ export const AwardEditCard = ({ award = emptyAward }: AwardEditCardProps) => {
         }`,
       };
     });
-    setAllUsers(users);
-
-    // Initializing a new image
-    if (isEmpty(award.image)) {
-      const reader = new FileReader();
-      reader.onloadend = () => {
-        setAwardImage(reader.result?.toString() ?? '');
-      };
-
-      const newImage = `https://picsum.photos/100/50/?original=${random(
```

Comment:
i got rid of the auto image generation since it doesn’t work as well with the new model, and i don’t think it’s *super* important